src/styles/routingTransitions.ts

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/core'
+import { transitions, sizes } from '@/shared/theme'
+
+export const routingTransitions = css`
+  .${transitions.names.fadeAndSlide}-enter {
+    opacity: 0;
+
+    & .${transitions.names.slide} {
+      transform: translateY(${sizes(8)});
+    }
+  }
+
+  .${transitions.names.fadeAndSlide}-enter-active {
+    opacity: 1;
+    & .${transitions.names.slide} {
+      transform: translateY(0);
+      transition: ${transitions.timings.routing} ${transitions.easing};
```

Comment:
Let's add an additional easing specifically for routing. Let's try `cubic-bezier(0.87, 0, 0.13, 1)` for now